docs/babel.config.js

```diff
@@ -4,7 +4,7 @@ const fse = require('fs-extra');
 const errorCodesPath = path.resolve(__dirname, './public/static/error-codes.json');
 
 const { version: transformRuntimeVersion } = fse.readJSONSync(
-  require.resolve('@babel/runtime-corejs2/package.json'),
+  require.resolve('@babel/runtime-corejs3/package.json'),
```

Comment:
This is already on master, isn't it?

Reply:
Yes, I added it here to validate it first and because the docs/demos were broken without it before the other PR got merged